public/src/SampleData.js

```diff
@@ -32,23 +32,30 @@ var settings = {
 }
 
 // The rest is collections and can be stored in firestore
-var personCollection = [
+const personCollection = [
   {
     login: firebaseUserId, // Only for people with a login
     name: 'Test User',
     phone: '+62 123 123 123',
     address: 'Jalan Batu Mejan No. 88, Canggu, Kuta Utara, Kabupaten Badung, Bali 80361',
     email: 'some@address.com',
-    geolaction: {
+    geolocation: {
       latitude: '8.39111',
       longitude: '115.07361'
     },
-    type: 'employee | client | community manager | facility manager | super admin | village guy',
+    type: {
+      employee:false,
+      client:false,
+      community_manager:false,
+      facility_manager:false,
+      super_admin: true
```

Comment:
note a fan of this change as noted in the previous comment

Reply:
we want camelCase 👍